faunadb/deserialization_test.go

```diff
@@ -496,6 +496,18 @@ func TestDeserializeComplexStruct(t *testing.T) {
 	require.Equal(t, expected, object)
 }
 
+func TestDeserializeStructWithOmitEmptyTags(t *testing.T) {
+	type object struct {
+		Name string `fauna:"name,omitempty"`
+		Age  int64  `fauna:"age,omitempty"`
+	}
+
+	var obj object
+
+	require.NoError(t, decodeJSON(`{ "name": "John", "age": 0 }`, &obj))
```

Comment:
what would happen if you omit "age" in the json? will it be parse to 0?

Reply:
It should be skipped.